components/inputs/docs/input-date-time.md

```diff
@@ -10,13 +12,37 @@ size:xlarge
   import '@brightspace-ui/core/components/inputs/input-date.js';
   import '@brightspace-ui/core/components/inputs/input-time.js';
 </script>
+<script>
+  window.addEventListener('load', function () {
+    const demoElem = document.querySelector('#demo-element');
+    if (!demoElem.hasAttribute('data-first-load')) return;
+
+    setTimeout(function() {
+      var input = document.querySelector('d2l-input-date');
+      input.opened = true;
+    }, 1000);
```

Comment:
I think this should be 100

Reply:
This one's purposefully 1000 in order to emphasize the opening